ts/features/itwallet/discovery/components/ItwNfcNotSupportedComponent.tsx

```diff
@@ -0,0 +1,70 @@
+import { useMemo } from "react";
+import I18n from "../../../../i18n";
+import {
+  OperationResultScreenContent,
+  OperationResultScreenContentProps
+} from "../../../../components/screens/OperationResultScreenContent";
+import { useAvoidHardwareBackButton } from "../../../../utils/useAvoidHardwareBackButton";
+import { useItwDisableGestureNavigation } from "../../common/hooks/useItwDisableGestureNavigation";
+import { useIOSelector } from "../../../../store/hooks";
+import { itwIsWalletInstanceRemotelyActiveSelector } from "../../common/store/selectors/preferences";
+import { useIONavigation } from "../../../../navigation/params/AppParamsList";
+import { ITW_ROUTES } from "../../navigation/routes";
+
+export const ItwNfcNotSupportedComponent = () => {
+  const navigation = useIONavigation();
+
+  const isWalletInstanceRemotelyActive = useIOSelector(
```

Comment:
Is this selector correct? It is returning `false` even when I have a valid wallet instance, so if I mock this scenario:
- "Documenti su IO" -> on
- NFC -> disabled

I get the `ItwNfcNotSupportedComponent`, but I can activate "Documenti su IO".

Looking at that selector, I see its value is defined as `!remoteWalletInstanceStatus.is_revoked && !isItwValidLocally`.

Reply:
Addressed in https://github.com/pagopa/io-app/pull/6981/commits/cd6ef4c7bb760bccfba4d940c332473100d0f5ec